app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb

```diff
@@ -1,37 +1,29 @@
-<p>
-  Hey <%= @user.name %>!
-</p>
-<p>
-  Thanks for volunteering to be a <%= community_name %> tag moderator. We’ve added you as a moderator for <a href="<%= tag_url(@tag) %>">#<%= @tag.name %></a>. Note that your profile picture is now visible on this page.
-</p>
-<p>
-  As a tag moderator, you are able to:
-</p>
+<p>Hey <%= @user.name %>!</p>
+
+<p>Thanks for volunteering to be a <%= community_name %> tag moderator. We <%= link_to "##{@tag.name}", tag_url(@tag) %> . Note that your profile picture is now visible on this page.</p>
+
+<p>With great power comes great... you know the saying. As a tag moderator, you are able to:</p>
+
 <ul>
-  <li>
-     Remove #<%= @tag.name %> if it’s been added to an irrelevant post.
-  </li>
-  <li>
-    Add #<%= @tag %> to posts if applicable.
-  </li>
-  <li>
-    Update the #<%= @tag %> tag landing page, including submission guidelines and design visible in the sidebar.
-  </li>
+  <li>Remove #<%= @tag %> if it’s been added to an irrelevant post. Only remove tags when they do not fit well for the subject matter of the post. Do add more relevant tags if it will elevate the visibility of the post or make it discoverable to other audiences.</li>
+  <li>Add #<%= @tag %> to posts if applicable. This helps the best content become more easily discoverable!</li>
+  <li>Update the #<%= @tag %> tag landing page, including submission guidelines and design visible in the sidebar.</li>
```

Comment:
I find it surprising that these lines are just `@tag` when line 3 is `@tag.name` — I would have expected them to have similar semantics, but maybe it works anyway?

Reply:
This is a VERY good point, I just doubled checked this and in the view `@tag.name` and `@tag` both output the tag name. However, @tag in the method is the actual tag record. 

```
    @user = params[:user]
    @tag = params[:tag]
    @channel_slug = params[:channel_slug]

    subject = I18n.t("mailers.notify_mailer.moderator", tag_name: @tag.name)
    mail(to: @user.email, subject: subject)
  end
```

🧐 going to sleuth a bit more...